cli/ballerina-cli/src/main/resources/cli-help/ballerina-bindgen.help

```diff
@@ -13,32 +13,34 @@ DESCRIPTION
        Generate Ballerina bridge code for Java APIs.
 
        Ballerina bindings could be generated for Java classes residing inside Java libraries
-       or for standard Java classes. The Java classes will be mapped to the Ballerina
-       classes providing a seamless Java interoperability developer experience to Ballerina users.
+       or standard Java classes. The Java classes will be mapped onto Ballerina
+       objects making the developer experience of Ballerina Java interoperability seamless.
```

Comment:
Why change the phrasing here? "Ballerina objects" and "Ballerina classes" have 2 very different meanings. @IrushiL Please confirm the correct term to be used here.

Reply:
Seems like these changes were added when resolving the conflict caused by https://github.com/ballerina-platform/ballerina-lang/pull/31191/files. @IrushiL - Please confirm.

Reply:
It should be `Ballerina classes`.

In the conflicting PR, we have replaced the outdated term with the new one (some time ago we generated Ballerina objects now we generate Ballerina classes).

Reply:
Updated as "Ballerina classes".